audio/ocenaudio-bin/ocenaudio-bin.SlackBuild

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+# Slackware build script for ocenaudio-bin
+
+# Copyright 2025 Ioannis Anagnostakis GR
+# All rights reserved.
+#
+# Redistribution and use of this script, with or without modification, is
+# permitted provided that the following conditions are met:
+#
+# 1. Redistributions of this script must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE AUTHOR "AS IS" AND ANY EXPRESS OR IMPLIED
+#  WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO
+#  EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+#  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+#  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+#  OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+#  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+cd $(dirname $0) ; CWD=$(pwd)
+
+PRGNAM=ocenaudio-bin
+VERSION=${VERSION:-3.14.10}
+BUILD=${BUILD:-1}
+TAG=${TAG:-_SBo}
+PKGTYPE=${PKGTYPE:-tgz}
+
+SRCNAM64=ocenaudio_debian10
+SRCNAM32=
+COMPRESS=deb
+
+if [ -z "$ARCH" ]; then
+  case "$( uname -m )" in
+    i?86) ARCH=i586 ;;
+       *) ARCH=$( uname -m ) ;;
+  esac
+fi
+
+if [ ! -z "${PRINT_PACKAGE_NAME}" ]; then
+  echo "$PRGNAM-$VERSION-$ARCH-$BUILD$TAG.$PKGTYPE"
+  exit 0
+fi
+
+if [ "$ARCH" = "i586" ]; then
+  SRCNAM=$SRCNAM32
+elif [ "$ARCH" = "i686" ]; then
+  SRCNAM=$SRCNAM32
+elif [ "$ARCH" = "x86_64" ]; then
+  SRCNAM=$SRCNAM64
+else
+  SRCNAM=
+fi
+
+if [ -z "$SRCNAM" ] || [ -z "$COMPRESS" ]; then
+  echo "SRCNAM and/or COMPRESS are empty, exiting."
+  exit 1
+fi
+
+TMP=${TMP:-/tmp/SBo}
+PKG=$TMP/package-$PRGNAM
+OUTPUT=${OUTPUT:-/tmp}
+
+
+rm -rf $PKG
+mkdir -p $TMP $PKG $OUTPUT
+
+
+if [[ $COMPRESS = "deb" ]]; then
+# extract directly to $PKG/
+ ar x $SRCNAM.$COMPRESS
```

Comment:
This is broken - the source is in $CWD

Reply:
```sh
curl -v -o /dev/null https://www.ocenaudio.com/downloads/index.php/ocenaudio_debian10.deb 2>&1 | grep -i filename
< Content-Disposition: attachment; filename="ocenaudio_debian10_3.14.10.deb"
```

Reply:
The script should handle both names - when downloading with a browser etc it will get the content-disposition name. With wget etc it will have the last part of the URL as the filename